index.d.ts

```diff
@@ -44,7 +44,18 @@ export type DailyEvent =
   | 'network-connection'
   | 'fullscreen'
   | 'exited-fullscreen'
-  | 'error';
+  | 'error'
+  | 'click'
+  | 'mousedown'
+  | 'mouseup'
+  | 'mouseover'
+  | 'mousemove'
+  | 'touchstart'
+  | 'touchmove'
+  | 'touchend'
+  | 'live-streaming-started'
```

Comment:
Update the 'live-stream-*' events to correctly say 'live-streaming-*', and added them to the `DailyEvent` type. Our official docs need to follow suit!